station-renderer-arsenic/src/main/java/net/modificationstation/stationapi/impl/client/arsenic/renderer/render/BakedModelRendererImpl.java

```diff
@@ -196,7 +197,8 @@ private void renderBakedItemModelFlat(BakedModel model, ItemInstance stack, floa
         for (BakedQuad bakedQuad : model.getQuads(null, null, random)) {
             if (bakedQuad.getFace() != Direction.WEST) continue;
             int i = bl && bakedQuad.hasColor() ? this.itemColors.getColor(stack, bakedQuad.getColorIndex()) : -1;
-            i = colorF2I(redI2F(i) * brightness, greenI2F(i) * brightness, blueI2F(i) * brightness);
+            float light = MathHelper.lerp(bakedQuad.getEmission(), brightness, 1F);
```

Comment:
Same question as in `LightingCalculatorImpl`

Reply:
Same answer =)